src/executors.jl

```diff
@@ -365,13 +356,56 @@ function dispatch!(exec::Executor, graph::DispatchGraph; throw_error=true)
 
     len = length(graph.nodes)
     info(logger, "Executing $len graph nodes.")
-    res = asyncmap(f, 1:len; ntasks=div(len * 3, 2))
+
+    asyncmap(f1, 1:len; ntasks=div(len * 3, 2))
```

Comment:
It doesn't need to be asyncmap, a `for` loop will do just as well/better since we aren't actually waiting on tasks when calling reset so using async isn't doing anything here.